corehq/util/couch_doc_processor.py

```diff
@@ -147,6 +147,42 @@ def should_process(self, doc):
         return True
 
 
+class ProcessorProgressLogger(object):
+    def progress_starting(self, total, previously_visited):
+        print("Processing {} documents{}: ...".format(
```

Comment:
bleh... didn't notice the class below, although I still may prefer the default message parameter to a whole extra subclass.